disperser/apiserver/server.go

```diff
@@ -457,9 +463,10 @@ func (s *DispersalServer) checkRateLimitsAndAddRatesToHeader(ctx context.Context
 		// System Level
 		key := fmt.Sprintf("%s:%d-%s", systemAccountKey, param.QuorumID, SystemThroughputType.Plug())
 		requestParams = append(requestParams, common.RequestParams{
-			RequesterID: key,
-			BlobSize:    encodedSize,
-			Rate:        globalRates.TotalUnauthThroughput,
+			RequesterID:   key,
+			RequesterName: systemAccountKey,
```

Comment:
shouldn't these be `requesterName`?

Reply:
Hmm I see how it's confusing but in this case we are calculating the system rate limit metric so we want to make sure the metric that is being updated corresponds to the name "system".

Reply:
I think to Ian's point, this might be a good reason to name these something like `MetricsId` and `MetricsFlag`, so that the intended usage is clear to the library user. Not super critical though.